tests/cms/utils/test_cancel_translation.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from _pytest.logging import LogCaptureFixture
+    from django.test.client import Client
+    from pytest_django.fixtures import SettingsWrapper
+
+import pytest
+from _pytest.logging import LogCaptureFixture
+from django.test.client import Client
+from django.urls import reverse
+from pytest_django.fixtures import SettingsWrapper
+
+from integreat_cms.cms.constants import translation_status
+from integreat_cms.cms.models import Page
+from tests.conftest import ANONYMOUS, AUTHOR, EDITOR, MANAGEMENT, PRIV_STAFF_ROLES
+from tests.utils import assert_message_in_log
+
+IDS_PAGES_NOT_IN_TRANSLATION_PROCESS = [1, 2]
+IDS_PAGES_IN_TRANSLATION_PROCESS = [31, 32]
+LANGUAGE_SLUG = "en"
+REGION_SLUG = "augsburg"
+
+
+@pytest.mark.django_db
+def test_bulk_cancel_translation_process(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+    caplog: LogCaptureFixture,
+    settings: SettingsWrapper,
+) -> None:
+    client, role = login_role_user
+
+    # Test for english messages
+    settings.LANGUAGE_CODE = "en"
+
+    for page_id in IDS_PAGES_IN_TRANSLATION_PROCESS:
+        assert (
+            Page.objects.filter(id=page_id).first().get_translation_state(LANGUAGE_SLUG)
+            == translation_status.IN_TRANSLATION
+        )
+    for page_id in IDS_PAGES_NOT_IN_TRANSLATION_PROCESS:
+        assert (
+            not Page.objects.filter(id=page_id)
+            .first()
+            .get_translation_state(LANGUAGE_SLUG)
+            == translation_status.IN_TRANSLATION
+        )
+
+    kwargs = {"region_slug": REGION_SLUG, "language_slug": LANGUAGE_SLUG}
+    page_ids = IDS_PAGES_IN_TRANSLATION_PROCESS + IDS_PAGES_NOT_IN_TRANSLATION_PROCESS
```

Comment:
```suggestion
    combined_page_ids = IDS_PAGES_IN_TRANSLATION_PROCESS + IDS_PAGES_NOT_IN_TRANSLATION_PROCESS
```
Super nit: I'm not even sure if this actually makes it better, so feel free to reject this suggestion :) But while reading the test line 79 was the only one that wasn't immediately clear to me. But again, looking up what page_ids was took a few seconds and is definitely justifiable